ItemChanger.Core/ItemChangerProfile.cs

```diff
@@ -212,72 +232,115 @@ public void AddPlacement(
+        UnloadIfLoaded(existing);
+        return true;
+    }
 
-        if (placements.TryGetValue(placement.Name, out Placement? existP))
+    private bool MergeKeepingOld(Placement newPlacement, Placement existing)
+    {
+        existing.Items.AddRange(newPlacement.Items);
+        if (State >= LoadState.PlacementsLoadCompleted)
         {
-            switch (conflictResolution)
+            foreach (Item item in newPlacement.Items)
             {
-                case PlacementConflictResolution.MergeKeepingNew:
-                    placement.Items.AddRange(existP.Items);
-                    placements[placement.Name] = placement;
-                    if (State >= LoadState.PlacementsLoadCompleted)
-                    {
-                        existP.Unload();
-                    }
-                    break;
-                case PlacementConflictResolution.MergeKeepingOld:
-                    existP.Items.AddRange(placement.Items);
-                    if (State >= LoadState.PlacementsLoadCompleted)
-                    {
-                        foreach (Item item in placement.Items)
-                        {
-                            item.LoadOnce();
-                        }
-                    }
-                    break;
-                case PlacementConflictResolution.Replace:
-                    placements[placement.Name] = placement;
-                    if (State >= LoadState.PlacementsLoadCompleted)
-                    {
-                        existP.Unload();
-                    }
-                    break;
-                case PlacementConflictResolution.Ignore:
-                    break;
-                case PlacementConflictResolution.Throw:
-                default:
-                    throw new ArgumentException(
-                        $"A placement named {placement.Name} already exists"
-                    );
+                item.LoadOnce();
             }
         }
-        else
-        {
-            placements.Add(placement.Name, placement);
-        }
+        return false;
+    }
 
-        // if the final placement ending up in the profile is the newly added one, it may need to be loaded to catch up.
-        if (State >= LoadState.PlacementsLoadCompleted && placements[placement.Name] == placement)
+    private bool ReplacePlacement(Placement newPlacement, Placement existing)
+    {
+        placements[newPlacement.Name] = newPlacement;
+        UnloadIfLoaded(existing);
+        return true;
+    }
+
+    private void UnloadIfLoaded(Placement placement)
+    {
+        if (State >= LoadState.PlacementsLoadCompleted)
         {
-            placement.LoadOnce();
+            placement.Unload();
         }
     }
 
-    public void AddPlacements(
-        IEnumerable<Placement> placements,
-        PlacementConflictResolution conflictResolution = PlacementConflictResolution.MergeKeepingNew
-    )
+    private void CatchUpPlacement(Placement placement)
```

Comment:
I think the naming here should mirror above, like LoadIfNeeded (possibly with the above becoming UnloadIfNeeded)